business-registry-dashboard/app/pages/index.vue

```diff
@@ -308,22 +295,30 @@ const selectedStates = ref([])
                   v-if="isRejectedName(nrName)"
                   class="table-cell size-4 pr-1 align-top text-red-500"
                   name="i-mdi-close"
+                  aria-hidden="true"
                 />
                 <UIcon
                   v-if="isApprovedName(nrName)"
-                  color="green"
                   class="table-cell size-4 pr-1 align-top text-green-500"
                   name="i-mdi-check"
+                  aria-hidden="true"
                 />
-                <div class="table-cell pl-2 align-top font-semibold">{{ nrName.name }}</div>
-              </b>
-            </b>
-            <b
+                <span v-if="isApprovedName(nrName)" class="sr-only">{{ $t('table.affiliation.cell.name.approved', { name: nrName.name }) }}</span>
```

Comment:
add sr-only 'approved' or 'rejected' to table cell if its a name request, so sr users can know the status instead of only relying on the icon